TLM/TMPE.API/Traffic/Data/SpeedValue.cs

```diff
@@ -122,6 +122,14 @@ public KmphValue ToKmphPrecise()
         public static SpeedValue operator +(SpeedValue left, SpeedValue right)
             => new(left.GameUnits + right.GameUnits);
 
+        public static bool operator ==(SpeedValue left, SpeedValue right) {
```

Comment:
Structs like here will be passed by value, but the value is a float (4 bytes). Not sure how bad this is, hopefully no impact.